include/div64.h

```diff
@@ -231,6 +231,7 @@ extern u32 __div64_32(u64 *dividend, u32 divisor);
 
 #endif /* BITS_PER_LONG */
 
+#ifndef CONFIG_SEL4
```

Comment:
Whilst I don't have a counter-argument, it's not clear to me why we had to protect by adding `#ifndef CONFIG_SYS` here (and elsewhere).

Reply:
In this case `lldiv` is provided by the Musl C library so needed to excluded here to prevent a name clash.